client/environment.go

```diff
@@ -109,6 +110,42 @@ type EnvironmentCreate struct {
 	VcsCommandsAlias            string                `json:"vcsCommandsAlias"`
 }
 
+// When converted to JSON needs to be flattened. See custom MarshalJSON below.
+type EnvironmentCreateWithoutTemplate struct {
```

Comment:
The struct was defined in a manner to support future changes to the template and environment create payloads.